src/components/OrderPage/QuantityButtons/index.js

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react"
+import './QuantityButtons.scss'
+
+const QuantityButtons = ({ foodName, price, addItem, removeItem, orderObject }) => {
+    const [quantity, setQuantity] = useState(0)
+
+    function findQty() {
```

Comment:
Quantity is being used over Qty elsewhere, could you be more consistent? Either change all qty to quantity, or the other way around